packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -166,8 +166,10 @@ export function getInternalReactConstants(
   // **********************************************************
   // The section below is copied from files in React repo.
   // Keep it in sync, and add version guards if it changes.
-  if (gte(version, '17.0.0-alpha')) {
-    // TODO (Offscreen) Update the version number above to reflect the first Offscreen alpha/beta release.
+  //
+  // TODO Update the gt() check below to be gte() whichever the next version number is.
+  // Currently the version in Git is 17.0.2 (but that version has not been/may not end up being released).
+  if (gt(version, '17.0.1')) {
```

Comment:
We could probably leave the `gt()` check as-is, but I think `gte()` is better because it's more explicit. Also if we next released 17.1.0 next, and then went back and bug fix released 17.0.2 from a fork of 17.0.1 (unlikely but possible) this `gt()` check would fail.